style.css

```diff
@@ -158,7 +158,7 @@ body {
 
   
   /********** Story *************/
-  
+
   .bubbles{
     background-image: url("/Assignment/Images/bubbles.png");
```

Comment:
@SolarFlare66 

Just as a heads up - you may have problems linking these paths because they are using absolute URLS. 

If you create an `/images` folder in your repository here is how you would reference your image using a relative path: 

```css
background-image: url("images/bubbles.png");
```

Note the lack of starting `/` and removal of the assignments folder. 

Hope that helps when you drop in the images into your repo.

Reply:
I will investigate this. 
Thank you for the tip.